src/back/auth/src/api/helpers.cpp

```diff
@@ -23,4 +23,13 @@ std::string getCallerUrl(const server::http::HttpRequest& req, bool addApiPrefix
 	return scheme + "://" + host + apiPrefix;
 }
 
+std::string getCallerUserAgent(const server::http::HttpRequest& req)
+{
+	auto userAgent = req.GetHeader("User-Agent");
```

Comment:
У userver есть константы под популярные хидеры. Они советуют использовать их.
```suggestion
	auto userAgent = req.GetHeader(http::headers::kUserAgent);
```

Reply:
поменял, использую константу фреймворка.